executor/adapter.go

```diff
@@ -263,10 +263,15 @@ func (a *statement) logSlowQuery() {
 		sql = sql[:cfg.Log.QueryLogMaxLen] + fmt.Sprintf("(len:%d)", len(sql))
 	}
 	connID := a.ctx.GetSessionVars().ConnectionID
+	logEntry := log.WithFields(log.Fields{
+		"connectionId": connID,
+		"costTime":     costTime,
+		"sql":          sql,
+	})
 	if costTime < time.Duration(cfg.Log.SlowThreshold)*time.Millisecond {
-		log.Debugf("[%d][TIME_QUERY] %v %s", connID, costTime, sql)
+		logEntry.Debugf("query")
```

Comment:
Why change `TIME_QUERY` to `query` ?

Reply:
I found it actually logs all queries, not just for slow queries.